docs/internals/deprecation.txt

```diff
@@ -1230,7 +1230,7 @@ details on these changes.
   instead use ``django.contrib.formtools.utils.form_hmac``
 
 * The function-based generic view modules will be removed in favor of their
-  class-based equivalents, outlined :doc:`here
+  class-based equivalents, outlined in :doc:`class-based views
```

Comment:
We could also add an anchor in `docs/topics/class-based-views/index.txt`
```diff
diff --git a/docs/topics/class-based-views/index.txt b/docs/topics/class-based-views/index.txt
index ec126099f6..fb934fae10 100644
--- a/docs/topics/class-based-views/index.txt
+++ b/docs/topics/class-based-views/index.txt
@@ -1,3 +1,5 @@
+.. _class-based-views-explanation:
+
 =================
 Class-based views
 =================

```

and use it here

```suggestion
  class-based equivalents, outlined in :ref:`class-based-views-explanation`.
```

and in other places:

- docs/releases/1.3.txt:``See :doc:`the documentation on class-based generic views</topics/class-based-views/index>` ``
- docs/releases/1.5.txt:``In all :doc:`generic class-based views </topics/class-based-views/index>` ``
- docs/topics/http/urls.txt:``   </topics/class-based-views/index>`). The view gets passed the following``
- docs/topics/auth/default.txt:``When using :doc:`class-based views </topics/class-based-views/index>`, you can``
- docs/topics/auth/default.txt:``    When using :doc:`class-based views </topics/class-based-views/index>`, you``
- docs/topics/auth/default.txt:``All views are :doc:`class-based </topics/class-based-views/index>`, which allows``
- docs/internals/deprecation.txt:``  </topics/class-based-views/index>`.``
- docs/ref/class-based-views/index.txt:``:doc:`/topics/class-based-views/index` topic guide.``
- docs/glossary.txt:``        See :doc:`/topics/class-based-views/index`.``
- docs/index.txt:``  :doc:`Overview <topics/class-based-views/index>` |``
- docs/intro/tutorial04.txt:``</topics/class-based-views/index>`.``